### PR TITLE
Minor fixes to calculator bot

### PR DESCRIPTION
**Pull request by pyko**
Going through and saw some changes needed for the simplified calculator bot.

Btw, this one is much nicer than the other one :D

### CODE REVIEW

File: src/Chat/Bot/Calculator.hs [hunk 1]
```diff
@@ -6,6 +6,21 @@ import           Chat.Bot.Answer.Calculator
 import           Chat.Bot.Misc.Calculator
 import           Chat.Data
 
+{-
+  USAGE: /calculate a operation b
```
Comment (charleso) [inline src/Chat/Bot/Calculator.hs:10]: Whoops. So I hadn't added this yet. Thanks @pyko

File: src/Chat/Bot/Calculator.hs [hunk 2]
```diff
@@ -25,15 +40,15 @@ calculate op a b =
 
 
 
--- LEVEL: Easy
+-- | LEVEL: Easy
```
Comment (charleso) [inline src/Chat/Bot/Calculator.hs:43]: Whoops again

**Comment by charleso** [comment]
Nice!

File: src/Chat/Bot/Calculator.hs (hunk 1 above)
Comment (pyko) [inline src/Chat/Bot/Calculator.hs:10]: No worries, second pair of eyes always useful :)

File: src/Chat/Bot/Calculator.hs (hunk 2 above)
Comment (pyko) [inline src/Chat/Bot/Calculator.hs:43]: Hehe, now I  know a innocent bar actually is important (was wondering why my tests weren't running...)

Comment (charleso) [inline src/Chat/Bot/Calculator.hs:43]: > Hehe, now I know a innocent bar actually is important 

One of the (many) reasons why doctests are silly. I am/was sorely tempted just to make them normal tests.

**Pull request merged**